src/main/java/edu/wpi/first/outlineviewer/view/dialog/PreferencesDialog.java

```diff
@@ -9,7 +10,9 @@
 /**
  * A dialog for changing the app preferences.
  */
-public class PreferencesDialog extends Dialog<ButtonType> {
```

Comment:
Right now, the only usages are "Cancel, Ok" and "Quit, Start".  Both answer the boolean question "Should we save/start?"